plugins/usd/module/vtkF3DUSDImporter.h

```diff
@@ -40,6 +40,15 @@ class vtkF3DUSDImporter : public vtkF3DImporter
    */
   vtkSetMacro(FileName, std::string);
 
+  /**
+   * Get the level of animation support in this importer, which is always
+   * AnimationSupportLevel::UNIQUE
+   */
+  AnimationSupportLevel GetAnimationSupportLevel() override
```

Comment:
Not covered? I'm surprised we do not have a USD animation test.

Reply:
turns out coverage was correct. this was not tested because we do not check the animation support level unless we need to and just testing an animated file without specifying animation-indices do not require checking animation support level.

I've added a test.